test/conf/karma-common.conf.js

```diff
@@ -14,8 +16,13 @@ var testFiles = [
 module.exports = function(config) {
     return {
         basePath: "../..",
-        frameworks: ["jasmine"],
-        files: sourceFiles.concat(testFiles),
+        frameworks: ["jasmine", 'commonjs'],
+        files: sourceFiles.concat(testFiles).concat(commonJsSourceFiles),
+        preprocessors: {
+        },
```

Comment:
It seems that karma fails to start with the empty `preprocessors` option (see https://travis-ci.org/vivliostyle/vivliostyle.js/builds/170985369#L746-L769).
Build a glob pattern from `commonJsSourceFiles` and use it as a key for the option?